web_ui/frontend/components/BooleanToggleButton.tsx

```diff
@@ -0,0 +1,50 @@
+import {Button, ButtonGroup, IconButton, Tooltip} from '@mui/material';
+import {Check, DoNotDisturb, HorizontalRule} from "@mui/icons-material";
+
+
+interface BooleanToggleButtonProps {
+    label: string;
+    value?: boolean;
+    onChange: (value?: boolean) => void;
+}
+
+export const BooleanToggleButton = ({label, value, onChange}: BooleanToggleButtonProps) => {
+    return (
+        <ButtonGroup
+            aria-label={label}
+        >
+            <Button disabled size={"small"}>
```

Comment:
Can we pass a brief description to the label as tooltips? At least `Pelican Server` sounds a bit confused to users not knowing the difference